service/service.go

```diff
@@ -0,0 +1,77 @@
+package service
+
+import (
+	"context"
+	"github.com/ONSdigital/dp-bulletin-api/api"
+	"github.com/ONSdigital/dp-bulletin-api/config"
+	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	"github.com/ONSdigital/go-ns/server"
+	"github.com/ONSdigital/log.go/log"
+	"github.com/gorilla/mux"
+	"os"
+	"os/signal"
+	"time"
+)
+
+// run the application
+func Run(buildTime,gitCommit,version string, args []string) error {
+	log.Namespace = "dp_bulletin_api"
+	cfg, err := config.Get()
+	ctx := context.Background()
+	if err != nil {
+		log.Event(ctx, "unable to retrieve service configuration", log.Error(err))
```

Comment:
I don't  think you need to log this here as you are returning the error - either log it and handle it or return it and let the caller make that decision. If the additional context will be useful maybe consider using `errors.Wrap()`: 

```
return errors.Wrap(err, "unable to retrieve service configuration")
```
The main func will log this out.